src/components/main/main.jsx

```diff
@@ -4,10 +4,20 @@ import {connect} from 'react-redux';
 import {ActionCreator} from '../../reducer/reducer.js';
 import FilmList from "../movie-list/movie-list.jsx";
 import FilmGenre from "../movie-genre/movie-genre.jsx";
+import BtnLoad from "../btn-load/btn-load.jsx";
 
 
 const Main = (props) => {
-  const {films, cardFilms, availableGenres, currentGenre, onTitleClick, onGenreClick, onFilmCardClick} = props;
+  const {films, cardFilms, availableGenres, currentGenre, onTitleClick, onShowMoreClick, onGenreClick, showFilms, onFilmCardClick} = props;
+  const showedFilms = films.slice(0, showFilms);
+  const getShowedMore = ()=> {
+    if (showedFilms.length === films.length) {
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
```suggestion
  const getShowedMore = () => {
    return showedFilms.length === films.length
  };
```
да и можно эту проверку инлайном сделать, без функции